src/ty.md

```diff
@@ -281,7 +281,7 @@ modules choose to import a larger or smaller set of names explicitly.
 Let's consider the example of a type like `MyStruct<u32>`, where `MyStruct` is defined like so:
 
 ```rust,ignore
-struct MyStruct<T> { x: u32, y: T }
+struct MyStruct<T> { x: str, y: T }
```

Comment:
This is not valid. Unsized types like `str` cannot be used here, they may only appear as the type of the very last field.

Please choose a different type.